Core/kinematics.py

```diff
@@ -112,10 +124,14 @@ def calcIK(self,Lp,angles,center):
         (Tlf,Trf,Tlb,Trb)= self.bodyIK(omega,phi,psi,xm,ym,zm)
 
         Ix=np.array([[-1,0,0,0],[0,1,0,0],[0,0,1,0],[0,0,0,1]])
-        return np.array([self.legIK(np.linalg.inv(Tlf)@Lp[0]),
-        self.legIK(Ix@np.linalg.inv(Trf)@Lp[1]),
-        self.legIK(np.linalg.inv(Tlb)@Lp[2]),
-        self.legIK(Ix@np.linalg.inv(Trb)@Lp[3])])
+        # return np.array([self.legIK(np.linalg.inv(Tlf)@Lp[0]),
+        # self.legIK(Ix@np.linalg.inv(Trf)@Lp[1]),
+        # self.legIK(np.linalg.inv(Tlb)@Lp[2]),
+        # self.legIK(Ix@np.linalg.inv(Trb)@Lp[3])])
+        return np.array([self.legIK(np.linalg.inv(Tlf).dot(Lp[0])),
+        self.legIK(Ix.dot(np.linalg.inv(Trf).dot(Lp[1]))),
+        self.legIK(np.linalg.inv(Tlb).dot(Lp[2])),
+        self.legIK(Ix.dot(np.linalg.inv(Trb).dotLp[3]))])
```

Comment:
self.legIK(Ix.dot(np.linalg.inv(Trb).dot(Lp[3])))])

Reply:
i fixed this on master.